Gems/Atom/Feature/Common/Code/Source/PostProcessing/BloomBlurPass.cpp

```diff
@@ -109,7 +109,7 @@ namespace AZ
 
             RPI::Scene* scene = GetScene();
             PostProcessFeatureProcessor* fp = scene->GetFeatureProcessor<PostProcessFeatureProcessor>();
-            RPI::ViewPtr view = scene->GetDefaultRenderPipeline()->GetDefaultView();
+            RPI::ViewPtr view = scene->GetDefaultRenderPipeline()->GetFirstView(GetPipelineViewTag());
```

Comment:
Maybe irrelevant to your PR, it probably shouldn't use GetDefaultRenderPipeline() but use m_pipeline